xtas/_defaultconfig.py

```diff
@@ -4,21 +4,21 @@
 
 # See http://docs.celeryproject.org/en/latest/configuration.html for the
 # allowed options in this dict.
-CELERY = dict(
-    BROKER_URL='amqp://',
-    CELERY_RESULT_BACKEND='amqp://',
+CELERY = {
+    'BROKER_URL': 'amqp://',
+    'CELERY_RESULT_BACKEND': 'amqp://',
 
-    CELERY_TASK_SERIALIZER='json',
-    CELERY_RESULT_SERIALIZER='json',
-    CELERY_ACCEPT_CONTENT=['json'],
-    CELERY_TIMEZONE='Europe/Amsterdam',
-    CELERY_ENABLE_UTC=True,
+    'CELERY_TASK_SERIALIZER': 'json',
+    'CELERY_RESULT_SERIALIZER': 'json',
+    'CELERY_ACCEPT_CONTENT': ['json'],
+    'CELERY_TIMEZONE': 'Europe/Amsterdam',
+    'CELERY_ENABLE_UTC': True,
 
-    CELERY_TASK_RESULT_EXPIRES=3600,
+    'CELERY_TASK_RESULT_EXPIRES': 3600,
 
     # Uncomment the following to make Celery tasks run locally (for debugging).
-    #CELERY_ALWAYS_EAGER=True,
-)
+    #'CELERY_ALWAYS_EAGER': True,
+}
```

Comment:
Actually, I really liked the name=value syntax here. This file is produced by make_config.py as a default configuration file, so it's expected to be edited.